cinema/views.py

```diff
@@ -56,3 +83,54 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+    @staticmethod
+    def _params_to_str(qs):
+        return [str(str_id) for str_id in qs.split(",")]
+
+    def get_queryset(self):
+        queryset = self.queryset
+        if self.action == "list":
+            queryset = self.queryset\
+                .select_related("movie", "cinema_hall")\
```

Comment:
The preferred style for line-breaking is using `()` instead of `\`. By the way, now it's on the checklist! :)
https://github.com/mate-academy/py-tickets-orders/blob/master/checklist.md#code-style